src/routes/balances.rs

```diff
@@ -0,0 +1,84 @@
+use std::str::FromStr;
+
+use rocket::{futures::future::join_all, http::Status, serde::json::Json};
+use solana_sdk::pubkey::Pubkey;
+
+use lib::{
+    balances::{
+        accounts::get_parsed_accounts,
+        dapps::{raydium::get_parsed_positions, types::ParsedPosition},
+        types::BalancesResponse,
+    },
+    client::SolanaMirrorClient,
+    utils::get_rpc,
+    Error,
+};
+
+#[get("/balances/<address>?<positions>")]
+pub async fn accounts_handler(
+    address: &str,
+    positions: Option<bool>,
```

Comment:
This should be a more generic param, we're going to have dapps that are not necessarily LPs